Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -687,15 +687,14 @@ private List<string> expandFileNames(string location)
                 location = location.Replace("~", homePath);
             }
 
+            string fileName = Path.GetFileName(location);
+            string directoryName = Path.GetDirectoryName(location);
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                location = Path.GetFullPath(location);
+                location = directoryName + "\\" + fileName;
```

Comment:
will \\ work on Linux/Mac?

Reply:
Yes, the connector was able to build and run tests on Linux and Mac machines as well